FogCreek/Glitch-Community
# Joyous nutria: 429s on inviting users to a team

## Links
* Remix link: https://joyous-nutria.glitch.me/
* Issue-tracker link: https://app.clubhouse.io/glitch/story/1702/weird-ui-messaging-in-response-to-a-team-invite-failing-due-to-rate-limiting

## GIF/Screenshots:
![Screen Shot 2019-11-22 at 11 42 17 AM](https://user-images.githubusercontent.com/6620164/69443877-43be9e80-0d1d-11ea-833e-6471e94bdace.png)

## Changes:
Check for 429s from the server for inviting/reinviting users and display a more meaningful error message, also stopped us from logging these events in sentry since it's not an exception.

I tried working on preventing showing the faded user avatar, but it looks like we get that data directly from the server.  

## How To Test:
* go to the remix and go to a team, try inviting the same person to it within a 5 min period you'll should get an error message about rate limiting. 

## Feedback I'm looking for:
anything
